### PR TITLE
test(e2e): debug e2e-tests, don't merge!

### PR DESCRIPTION
**Pull request by bendikp**
This PR add a collector on the flaky e2e-test. Related #19

### CODE REVIEW

**Review by erikgb — APPROVED** [review]

File: test/e2e/scenario/vulnerability-overflow/00-assert.yaml [hunk 1]
```diff
@@ -18,6 +18,9 @@ collectors:
     command: df -h
   - type: events
     namespace: image-scanner-jobs
+  - type: pod
+    namespace: image-scanner-jobs
+    selector: workload.statnett.no/name=vuln-app
```
Comment (erikgb) [inline test/e2e/scenario/vulnerability-overflow/00-assert.yaml:23]: I think we forgot to add these labels to the podTemplate in https://github.com/statnett/image-scanner-operator/pull/80. Maybe a dedicated fix PR to add that?

Comment (bendikp) [reply test/e2e/scenario/vulnerability-overflow/00-assert.yaml:23]: Yup, I think https://github.com/statnett/image-scanner-operator/pull/83 will fix this.

**Comment by erikgb** [comment]
Is this PR still in draft? Or ready for review?

**Comment by bendikp** [comment]
Still a draft, I didn't manage to make this a draft PR again. I think we have enough debug info from this PR in order to proceed down the rabbit hole, so I guess there are no need to merge this PR. Just keep it as a "debug PR".

**Pull request closed without merging**